velox/functions/sparksql/Register.cpp

```diff
@@ -119,6 +119,8 @@ void registerFunctions(const std::string& prefix) {
       prefix + "instr", instrSignatures(), makeInstr);
   exec::registerStatefulVectorFunction(
       prefix + "length", lengthSignatures(), makeLength);
+  registerFunction<SubstringIndexFunction, Varchar, Varchar, Varchar, int32_t>(
```

Comment:
Let's update doc for this function.
https://github.com/facebookincubator/velox/tree/main/velox/docs/functions/spark